### DISCUSSION TITLE
关于漏洞CVE-2022-26884

### DISCUSSION THREAD

**Discussion by DengSongRen** [apache/dolphinscheduler#12885] [Q&A] [answered]
请问下：关于CVE-2022-26884的解决描述
Users can read any files by log server, Apache DolphinScheduler users should upgrade to version 2.0.6 or higher.

实际2.0.6版本是哪个MR做的BUG修复？具体是什么原因导致

**Comment by coffeehb** [comment, marked as answer]
> 请问下：关于[CVE-2022-26884](https://github.com/advisories/GHSA-vpgf-fgm8-gxr2)的解决描述 Users can read any files by log server, Apache DolphinScheduler users should upgrade to version 2.0.6 or higher.
> 
> 实际2.0.6版本是哪个MR做的BUG修复？具体是什么原因导致

你的问题答案在这里吧。
https://github.com/apache/dolphinscheduler/issues/12528
